DataBase/db.go

```diff
@@ -0,0 +1,77 @@
+package database
```

Comment:
Let's name the directory using kebab-case: `dir-with-my-pkg`.
In this case use the name `database` or `db`